index.js

```diff
@@ -440,19 +486,43 @@ function collectObj (stream, cb) {
   function end () { cb(null, rows) }
 }
 
+// Object, (k, v -> v) -> Object
 function mapObj (obj, fn) {
   Object.keys(obj).forEach(function (key) {
     obj[key] = fn(key, obj[key])
   })
   return obj
 }
 
-function filterObj (obj, fn) {
-  var res = {}
-  Object.keys(obj).forEach(function (key) {
-    if (fn(key, obj[key])) {
-      res[key] = obj[key]
+// KdbPoint -> OsmVersion
+function kdbPointToVersion (pt) {
+  return pt.value.toString('hex')
+}
+
+// OsmDb, OsmVersion -> [OsmId]
+function getWayNodesOfDeletedDoc (osm, version, cb) {
```

Comment:
This has to be left to downstream. A deleted way could have two parents, each with different nodes. Deleted ways should be returned without nodes/refs. Same with relations.